certbot-apache/certbot_apache/configurator.py

```diff
@@ -138,6 +138,7 @@ def __init__(self, *args, **kwargs):
         self._enhance_func = {"redirect": self._enable_redirect,
                               "ensure-http-header": self._set_http_header,
                               "staple-ocsp": self._enable_ocsp_stapling}
+        self._skeletons = {}
```

Comment:
`_skeletons` is left over from the PRs this was built on and is no really longer used outside of tests. We should probably remove it.